src/common/koneapi.ts

```diff
@@ -447,3 +451,79 @@ export async function waitForResponse(
     webSocketConnection.prependListener('message', onMessage)
   })
 }
+
+/**
+ * Function to fetch basic information of an equipment
+ * @param accessToken valid access token
+ * @param equipmentIdWithPrefix equipment identifier with ken prefix
+ */
+export async function fetchEquipmentBasicInformation(accessToken: AccessToken, equipmentIdWithPrefix: string): Promise<EquipmentInfo> {

```

Comment:
Should these be split into a different file?

Reply:
IMHO yes

Reply:
Also, these demos are lacking on the error-catching side quite a bit. Maybe try-catch around the axios requests?